### PR TITLE
Clean up the PG.pl POD.

### PR DESCRIPTION
**Pull request by drgrice1**
All POD in the file is moved up to where each method is defined.  This ensures that the order of the generated html is consistent with the order in the file, and makes it easier to maintain the POD.  Methods are now `head2`'s instead of items so that they are in the index.

The POD in this file was not cleaned up in #836, so this finishes that task.

### CODE REVIEW

**Review by Alex-Jordan — APPROVED** [review]

File: macros/PG.pl [hunk 1]
```diff
@@ -185,18 +360,19 @@ ()
...
 
-If external is 1, it is assumed the full url is provided.  If external is 0 or
-not given, then file name will be prefixed with the webwork2/htdocs/ directory.
+If external is 1, it is assumed the full URL is provided. If external is 0 or
+not given, then file name will be served from the C<pg/htdocs> directory (if
+found).
 
 Additional attributes can be passed as a hash reference in the optional third
-argument.  These attributes will be added as attributes to the script tag.
+argument. These attributes will be added as attributes to the script tag.
```
Comment (pstaabp) [inline macros/PG.pl:375]: This existed before, but I think 5 lines down should have a `http:` or `https:` in front of the URL.

**Comment by pstaabp** [comment]
I also see an error in the POD generation: 

```
Around line 818:
Expected '=item *'
```

File: macros/PG.pl (hunk 1 above)
Comment (drgrice1) [reply macros/PG.pl:375]: I changed this to the link currently used by AppletObjects.pj which does include `https:`.  Thanks for pointing this out.

**Comment by drgrice1** [comment]
I also fixed the POD error.  That one was a last minute addition, and I was sloppy.

**Review by pstaabp — APPROVED** [review]
Looks good now.

**Pull request merged**